openapi/v2/core-data.yaml

```diff
@@ -159,9 +147,7 @@ components:
           items:
             $ref: '#/components/schemas/BaseReading'
       required:
-        - apiVersion
         - deviceName
-        - id
         - readings
```

Comment:
Responses don't need required since no validation is performed. Can remove `required` section from all responses.

Reply:
As the `Event` DTO model is also used in `AddEventRequest` included in this PR, probably should not remove the required section?

Reply:
Right...